content/en/docs/contribute/review/reviewing-prs.md

```diff
@@ -69,35 +72,50 @@ class third,fourth white
-      a summary of your review (and leave some positive comments for the contributor!),
-      approve the PR, comment or request changes as needed. New contributors should always
-      choose **Comment**.
+1. Go to
+   [https://github.com/kubernetes/website/pulls](https://github.com/kubernetes/website/pulls).
+   You see a list of every open pull request against the Kubernetes website and docs.
+
+2. Filter the open PRs using one or all of the following labels:
+
+   - `cncf-cla: yes` (Recommended): PRs submitted by contributors who have not signed the CLA cannot be merged.
+     See [Sign the CLA](/docs/contribute/new-content/overview/#sign-the-cla) for more information.
+   - `language/en` (Recommended): Filters for english language PRs only.
+   - `size/<size>`: filters for PRs of a certain size. If you're new, start with smaller PRs.
+
+   Additionally, ensure the PR isn't marked as a work in progress.
+   PRs using the `work in progress` label are not ready for review yet.
+
+3. Once you've selected a PR to review, understand the change by:
+
+   - Reading the PR description to understand the changes made, and read any linked issues
+   - Reading any comments by other reviewers
+   - Clicking the **Files changed** tab to see the files and lines changed
+   - Previewing the changes in the Netlify preview build by scrolling to the PR's build check
+     section at the bottom of the **Conversation** tab.
+     Here's a screenshot (this shows GitHub's desktop site; if you're reviewing
+     on a tablet or smartphone device, the GitHub web UI is slightly different):
+     {{< figure src="/images/docs/github_netlify_deploy_preview.png" alt="GitHub pull request details including link to Netlify preview" >}}
+     To open the preview, click on the  **Details** link of the **deploy/netlify** line in the
+     list of checks.
+
+4. Go to the **Files changed** tab to start your review.
+
+   - Click on the `+` symbol  beside the line you want to comment on.
+   - Fill in any comments you have about the line and click either **Add single comment**
+      (if you have only one comment to make) or  **Start a review** (if you have multiple comments to make).
+   - When finished, click **Review changes** at the top of the page. Here, you can add
+     a summary of your review (and leave some positive comments for the contributor!),
+     approve the PR, comment or request changes as needed. Please always use the "Comment"
+     option when finishing your review.
+
+     - Avoid clicking the "Request changes" button when finishing your review.
+       If you want to block a PR from being merged before some further changes are made, you can leave a comment "/hold".
+       The "Request changes" status is sticky and it can only be discarded by privileged contributors.
+
+     - Avoid clicking the "Approve" button when finishing your review.
+       Leaving a "/lgtm" comment is sufficient most of the time.
```

Comment:
**To reviewers:**

These are the main changes for this PR.
All other changes are about indentation and long line wrapping.